package/rmx-only/package

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(rm1-only rm2-only)
+pkgdesc="Ensure packages are not installed on the wrong device"
+pkgver=0.1.0-1
+url=http://0.0.0.0/
```

Comment:
Do we have a better way for expressing this?

Reply:
It might make more sense to use https://remarkable.com